boards/shields/adafruit_2_8_tft_touch_v2/adafruit_2_8_tft_touch_v2_nano.overlay

```diff
@@ -16,9 +17,9 @@
 		#gpio-cells = <2>;
 		gpio-map-mask = <0xffffffff 0xffffffc0>;
 		gpio-map-pass-thru = <0 0x3f>;
-		gpio-map = <10 0 &arduino_nano_header 4 0>,  /* D4  */
-			   <15 0 &arduino_nano_header 10 0>, /* D10 */
-			   <16 0 &arduino_nano_header 9 0>;  /* D9  */
+		gpio-map = <ARDUINO_HEADER_R3_D4 0 &arduino_nano_header 4 0>,
+			   <ARDUINO_HEADER_R3_D9 0 &arduino_nano_header 10 0>,
+			   <ARDUINO_HEADER_R3_D10 0 &arduino_nano_header 9 0>;
```

Comment:
Typo?
```suggestion
			   <ARDUINO_HEADER_R3_D9 0 &arduino_nano_header 9 0>,
			   <ARDUINO_HEADER_R3_D10 0 &arduino_nano_header 10 0>;

Reply:
@etienne-lms it looks like either the original comment or configuration was wrong because it says clearly in the binding that "15" is D9 and "16" is D10, which is not matching the original comment... so this PR is not changing anything functionally about this overlay

Reply:
Agree with @decsny here, that this commit is not changing anything functionally. D9 and D10 in comments (and after `&arduino_nano_header`) are referring to Arduino Nano connector. It was like that even with the commit that first introduces this mapping (Arduino Nano 33 BLE specific at the time) in https://github.com/zephyrproject-rtos/zephyr/commit/c203b033ef3696b717cef2f8ec41ed30fe7e48b8.

Reply:
Ok, thanks for the answer (and sorry for the late reply).
I'll remove the DNM label.